docs/backend-requests/resources/session-tokens.mdx

```diff
@@ -11,18 +11,18 @@ When a user is authenticated in your application, a short-lived session token is
 
 Below are the default session claims that Clerk generates for you:
 
-* iss
-* sub
-* sid
-* exp
-* nbf
-* iat
-* azp
-* act
-* orgs
-* org_id
-* org_slug
-* org_role
+* `azp`: authorized party - the url of the application that this token is authorized for.
```

Comment:
Just to elaborate, this will actually contain the value of the `Origin` header that was included in the original FAPI request made from the user agent. Most commonly it will be the URL of the application, as you state here, e.g. `https://example.com`.